frontend/frontend/settings.py

```diff
@@ -75,6 +77,68 @@
 
 WSGI_APPLICATION = 'frontend.wsgi.application'
 
+# Sentry integration
+RAVEN_CONFIG = {
+    # Set the Sentry API key here
+    'dsn': 'https://d9bec7e9f54943a281d5271c29932e7c:b57cfbbc5edc456aa2ece299cabbd785@sentry.io/104123',
```

Comment:
It would be interesting to try this out too:
```
    # If you are using git, you can also automatically configure the
    # release based on the git info.
    'release': raven.fetch_git_sha(os.path.dirname(__file__)),
```

From here: https://docs.sentry.io/clients/python/integrations/django/#setup